apps/agent/package.json

```diff
@@ -2,24 +2,20 @@
   "name": "@aragon/apps-agent",
   "version": "2.1.0",
   "scripts": {
-    "compile": "truffle compile",
-    "build": "cd app && npm run build",
-    "apm:prepublish": "npm run compile",
-    "apm:publish:major": "aragon apm publish major --files public/ --prepublish-script apm:prepublish",
-    "apm:publish:minor": "aragon apm publish minor --files public/ --prepublish-script apm:prepublish",
-    "apm:publish:patch": "aragon apm publish patch --files public/ --prepublish-script apm:prepublish",
+    "console": "buidler console",
+    "compile": "buidler compile --force",
+    "devchain": "buidler node",
+    "build": "cd app && yarn build",
     "lint": "solium --dir ./contracts",
-    "test": "TRUFFLE_TEST=true npm run ganache-cli:test",
-    "test:gas": "GAS_REPORTER=true npm test",
-    "coverage": "npx truffle run coverage",
-    "ganache-cli:test": "./node_modules/@aragon/contract-helpers-test/scripts/ganache-cli.sh",
-    "abi:extract": "truffle-extract --output abi/ --keys abi",
```

Comment:
Right, we may want to replace this with something else. Not sure if there's a buidler plugin to do this, but it would be great to move the ABI out of `artifacts`.

Reply:
Maybe we can update it to be a super simple plugin? Otherwise, if I update this line: https://github.com/sohkai/truffle-extract/blob/master/index.js#L36 with `npx buidler compile` the package keep working fine. I don't have a strong preference, do you?

Reply:
I decide to fork your package and publish a new version for buidler instead: https://github.com/0xGabi/buidler-extract

Reply:
Sounds good! Honestly not sure if this fits well as a small buidler plugin, but I can see it being useful!

I was thinking about this earlier, and in reality, we may actually want to have a tool that lets us configure what we publish. For most apps, there's a lot of extra fluff in the artifacts (e.g. Voting only needs `Voting.json`, not all the other artifacts).